dlls/winevulkan/vulkan.c

```diff
@@ -896,6 +896,24 @@ static void wine_vk_instance_free(struct wine_instance *instance)
     free(instance);
 }
 
+VkResult wine_vkSetLatencySleepModeNV(VkDevice device, VkSwapchainKHR swapchain, const VkLatencySleepModeInfoNV *pSleepModeInfo)
+{
+    VkLatencySleepModeInfoNV sleep_mode_info_host;
+
+    struct wine_device* wine_device = wine_device_from_handle(device);
+    struct wine_swapchain* wine_swapchain = wine_swapchain_from_handle(swapchain);
+
+    wine_device->low_latency_enabled = pSleepModeInfo->lowLatencyMode;
```

Comment:
According to [Vulkan Docs](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkSetLatencySleepModeNV.html):

> **Parameters**
> * `pSleepModeInfo` is `NULL` or a pointer […]

> **Description**
> If `pSleepModeInfo` is `NULL`, `vkSetLatencySleepModeNV` will disable low latency mode, low latency boost, and set the minimum present interval previously specified by [VkLatencySleepModeInfoNV](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkLatencySleepModeInfoNV.html) to zero on `swapchain`.

Shouldn't this function handle cases where `pSleepModeInfo` is `NULL`? Right now it looks like it will try to unconditionally dereference passed pointer without checking for `NULL` and likely crash if one was passed by the caller.